src/components-styled/landelijke-toelichting.tsx

```diff
@@ -0,0 +1,54 @@
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import ExternalLinkIcon from '~/assets/external-link.svg';
+import siteText from '~/locale';
+import theme from '~/style/theme';
+import { Box } from './base';
+import { ExternalLink } from './external-link';
+import { Tile } from './tile';
+import { Heading, Text } from './typography';
+
+export function LandelijkeToelichting() {
+  return (
+    <Tile
+      css={css({
+        bg: theme.colors.contextualContent,
+        flexDirection: [null, 'row'],
+      })}
```

Comment:
I believe Tile extends the Box component which means it should be possible to set props for `bg` and `flexDirecrion`
```suggestion
    <Tile
        bg={theme.colors.contextualContent}
        flexDirection={[null, 'row']}
```

Reply:
I tried those and that didn't work, the flexDirection was completely ignored...

Reply:
Oh that's a shame, I think moving the Tile's default bg css style to the `attrs()` should fix that:
```
export const Tile = styled(Box).attrs({ as: 'article', bg: 'white' })(
  css({
    display: 'flex',
    flexDirection: 'column',
    p: 4,
    borderRadius: 1,
    boxShadow: 'tile',
  })
);
```